packages/models/src/order/reOrder.interface.ts

```diff
@@ -6,4 +9,23 @@ export const enum ErrorMessageReOrder{
     CANNOT_CLEAR_CART = 'CANNOT CLEAR THE EXISTING ITEMS IN YOUR CART',
     ALL_ITEMS_INVALID = 'THESE ITEMS ARE NOT AVAILABLE RIGHT NOW',
     CANNOT_ADD_ITEMS = 'CANNOT ADD ITEMS TO THE CART'
-}
\ No newline at end of file
+}
+export interface ReOrderData {
+    id?: string;
+    userId?: string;
+    products?: IOrderProduct[] | null;
+    reDirectHome?: boolean;
```

Comment:
why this flag (reDirectHome) is needed?

Reply:
this flag is needed for frontend as they requested